react-components/src/components/RevealToolbar/measurementUrlStateHooks.ts

```diff
@@ -0,0 +1,83 @@
+/*!
```

Comment:
Can we move file to `hooks` folder

Reply:
Created a new `hooks` folder in the toolbar directory